docs-v2/integrate/guides/receive-webhooks-from-an-api.mdx

```diff
@@ -4,27 +4,32 @@ sidebarTitle: 'Receive webhooks from an API'
 description: 'Step-by-step guide on how to receive webhooks from an API.'
 ---
 
-[External API webhooks](/understand/concepts/webhooks) let you support real-time updates and are sometimes the only integration option.
+Nango makes it easy to add support for webhooks coming from **any** API. If your API doesn't have off-the-shelf support, simply request it in the [community](https://nango.dev/slack); we can deliver them in &lt;48h.
 
-# Check if Nango supports webhooks for your API
+Nango currently has off-the-shelf support for webhooks coming from the following APIs:
+- Github
+- Hubspot
+- Jira
+- Linear
+- Salesforce
+- Slack
 
-Nango has off-the-shelf support for webhooks coming from [this list of APIs](/reference/webhooks).
+# Configure external webhooks
 
-<Tip>
-The Nango team adds webhook support for new APIs in &lt;48h; simply ask in the [community](https://nango.dev/slack).
-</Tip>
+To send webhooks from an external API to Nango, configure webhooks on the external API portal, specifying the Nango URL to receive webhooks, which is accessible in your integration settings in the Nango UI.
 
-# Configure external webhooks
+# Configure webhooks from Nango
 
-To send webhooks from an external API to Nango, you need to configure webhooks on the external API portal. For this, you need the Nango URL receiving webhooks, which can be found in your integration settings. 
+When Nango receives a webhook from an external API, it notifies your app with another webhook. To set this up:
+1. go to the _Environment Settings_ tab in the Nango UI
+2. specify a Webhook URL to which Nango will send notifications
+3. listen for webhooks in your backend at the specified route
 
 # Forward external webhooks to your app
 
-Webhooks received by Nango from external APIs are immediately forwarded to your backend ([step-by-step guide](/integrate/guides/receive-webhooks-from-nango) to receive webhooks from Nango).
+Nango automatically forwards all webhooks from external APIs to your app. 
 
-Webhook forwarding is automatically enabled once you send webhooks from external APIs to Nango.  
-
-Here is the body of a webhook from Nango forwarding a webhook from an API: 
+Nango webhook forwards are POST requests with the following JSON body:
 ```json
 {
```

Comment:
Also has 

```
            connectionId,
            providerConfigKey
```

In some cases the connectionId may not be known if the validation fails or something goes wrong in identifying the webhook

Reply:
Done!